mlir/test/python/dialects/vector.py

```diff
@@ -51,10 +51,16 @@ def testTransferReadOp():
         with InsertionPoint(f.add_entry_block()):
             A, zero, padding, mask = f.arguments
             vector.TransferReadOp(
-                vector_type, A, [zero, zero], identity_map_attr, padding, mask=mask
+                vector_type,
+                A,
+                [zero, zero],
+                identity_map_attr,
+                padding,
+                [False, False],
+                mask=mask,
             )
             vector.TransferReadOp(
-                vector_type, A, [zero, zero], identity_map_attr, padding
```

Comment:
We're losing the default in Python? Can this be recovered somehow?

Reply:
> Can this be recovered somehow?

I don't think so.

From a quick scan, the Python bindings are build based on what's in _ODS_.  However, the default value for `in_bounds` is implemented by customising _C++_:
* the Op parsers (updated here), and 
* the Op builders (already flexible enough).

IIUC, such C++ customisation are not visible/available through Python bindings.